TemplePlus/ui/ui_pc_creation.cpp

```diff
@@ -684,6 +687,400 @@ Stat UiPcCreation::GetClassCodeFromWidgetAndPage(int idx, int page) {
 
 #pragma endregion
 
+//added stuff
+#pragma region Deity
+
+BOOL UiPcCreation::DeitySystemInit(UiSystemConf& conf) {
+	if (textureFuncs.RegisterTexture("art\\interface\\pc_creation\\buttonbox.tga", &buttonBox))
+		return 0;
+
+	for (int i = 0; i < DEITY_COUNT_NEW; i++) {
+		deityBtnMapping.push_back(i);
+	}
+	deityNames[0] = "None";
```

Comment:
Needs to read the strings from mes file, especially when rewriting legacy functions (i.e. not new strings).